tn_api_hotel.yml

```diff
@@ -1080,3 +1121,130 @@ components:
                 description: The full text of the ruledescription
                 example: 'non-refundable rate cancellation permitted up to 1days before arrival'
 
+    HotelBookRequest:
+      required:
+        - super_trip_id
+        - passengers
+        - payment_info
+        - hotel_bookings
+      properties:
+        super_trip_id:
+          type: string
+          example: "bdf876sf6sfsdf87sdf"
+          description: Id of the super trip.
+        passengers:
+          type: array
+          description: A list of passengers that will be booked to this hotel.
+          items:
+            type: object
+            required:
+              - first_name
+              - last_name
+              - date_of_birth
+              - gender
+              - passenger_type
+              - email
+              - phone_number
+            properties:
+              first_name:
+                description: First Name.
+                type: string
+                example: "Andres"
+              last_name:
+                description: Last Name.
+                type: string
+                example: "Collart"
+              date_of_birth:
+                description: Date of Birth.
+                type: string
+                format: date
+                example: "2017-07-23"
+              gender:
+                description: Gender
+                type: string
+                example: "M"
+                enum: [F, M]
+              passenger_type:
+                description: Type of passenger.
+                type: string
+                example: "ADT"
+                enum: [ADT, STU, CHD, INF]
+              email:
+                description: Passenger's email address.
+                type: string
+                example: "andres.collar@tripninja.io"
```

Comment:
hmm typo on my email!

Reply:
I copied it from somewhere else in the docs -- does it matter, just an example?